mos_tests/ironic/manage_instances_test.py

```diff
@@ -0,0 +1,162 @@
+        disk_format='raw',
+        container_format='bare',
+        hypervisor_type='baremetal',
+        cpu_arch='x86_64',
+        fuel_disk_info=json.dumps(settings.IRONIC_GLANCE_DISK_INFO))
+
+    if not image_file.file.closed:
+        src = urllib.request.urlopen(settings.IRONIC_IMAGE_URL)
+        with tarfile.open(fileobj=src, mode='r|gz') as tar:
+            img = tar.extractfile(tar.firstmember)
+            while True:
+                data = img.read(1024)
+                if not data:
+                    break
+                image_file.file.write(data)
+            image_file.file.close()
+    with open(image_file.name) as f:
+        os_conn.glance.images.upload(image.id, f)
+    src.close()
+
+    yield image
+    os_conn.glance.images.delete(image.id)
+
+
+@pytest.yield_fixture
+def ironic_node(baremetal_node, os_conn, ironic, server_ssh_credentials):
+
+    def get_image(name):
+        return os_conn.nova.images.find(name=name)
+
+    driver_info = {
+        'ssh_address': server_ssh_credentials['ip'],
+        'ssh_username': server_ssh_credentials['username'],
+        'ssh_key_contents': server_ssh_credentials['key'],
+        'ssh_virt_type': 'virsh',
+        'deploy_kernel': get_image('ironic-deploy-linux').id,
+        'deploy_ramdisk': get_image('ironic-deploy-initramfs').id,
+        'deploy_squashfs': get_image('ironic-deploy-squashfs').id,
+    }
+    properties = {
+        'cpus': baremetal_node.vcpu,
+        'memory_mb': baremetal_node.memory,
+        'local_gb': settings.IRONIC_DISK_GB,
+        'cpu_arch': 'x86_64',
+    }
+    node = ironic.node.create(driver='fuel_ssh', driver_info=driver_info,
+                              properties=properties)
+    mac = baremetal_node.interface_by_network_name('baremetal')[0].mac_address
+    port = ironic.port.create(node_uuid=node.uuid, address=mac)
+    yield node
+    instance_uuid = ironic.node.get(node.uuid).instance_uuid
+    if instance_uuid:
+        os_conn.nova.servers.delete(instance_uuid)
+        common.wait(lambda: len(os_conn.nova.servers.findall(
+                        id=instance_uuid)) == 0,
+                    timeout_seconds=60, waiting_for='instance to be deleted')
+    ironic.port.delete(port.uuid)
+    ironic.node.delete(node.uuid)
+
+
+@pytest.yield_fixture
+def instance(os_conn, ubuntu_image, flavor, keypair):
+    baremetal_net = os_conn.nova.networks.find(label='baremetal')
+    instance = os_conn.create_server('ironic-server', image_id=ubuntu_image.id,
+                                     flavor=flavor.id, key_name=keypair.name,
+                                     nics=[{'net-id': baremetal_net.id}],
+                                     timeout=60 * 10)
+    yield instance
+    instance.delete()
+
+
+@pytest.mark.check_env_('has_ironic_conductor')
+@pytest.mark.need_devops
+def test_instance_terminate(env, ironic, os_conn, ironic_node, ubuntu_image,
+                            flavor, keypair, env_name):
```

Comment:
You don't use `env_name` fixture in this test. Let's delete it (and check another fixture)